src/common/enums/join_type.cpp

```diff
@@ -38,4 +38,8 @@ bool IsRightOuterJoin(JoinType type) {
 	return type == JoinType::OUTER || type == JoinType::RIGHT;
 }
 
+bool IsInnerJoin(JoinType type) {
```

Comment:
nit: perhaps remove this function? It is unlikely we will add more inner joins later on.